test-suite/tests/compact-p002-out.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@context": {
+    "ex": {"@id": "http://example.org/"}
+  },
+  "@id": "http://example.org/id1",
+  "@type": ["http://example.org/Type1", "http://example.org/Type2"],
+  "http://example.org/term": {"@id": "http://example.org/id2"}
+}
```

Comment:
@gkellogg, ok, as an attempt to fix 1.0 I agree. Perhaps with 1.1 we can make it such that a colon must be present to define a prefix and only use the trailing gen-delim behavior in 1.0 for compatibility purposes?